src/frontend/js/widgets/Dashboard/components/DashboardSidebar/index.tsx

```diff
@@ -76,7 +76,10 @@ export const DashboardSidebar = ({
 
         <ul>
           {menuLinks.map((link) => (
-            <li key={link.to}>
+            <li
+              key={link.to}
+              className={selectedLink && selectedLink === link.to ? 'active' : undefined}
+            >
               <NavLink to={link.to}>{link.label}</NavLink>
```

Comment:
I detect a little misbehavior about the hover area of the link.
It should fill all the space available.Currently this is not the case so user can think it is hovering link and can click but in fact it is just hovering the li element.

COPY GIF HERE